doc/multiple_ports.rst

```diff
@@ -0,0 +1,88 @@
+Multiple ports
+==============
+
+
+Configuration of bridge using linux
+-----------------------------------
+
+Tested with Raspberry PI 3B+ and USB ethernet dongle USB3GIG.
+
+            +-------------+
+            |    br0      |
+            +------+------+
+            | eth0 | eth1 |
+            +------+------+
+
+
+p-net supports multiple ethernet ports. To use multiple ports the ports
+shall be grouped into a bridge. In such a configuration the profinet 
+interface is the bridge and the profinet ports are the ethernet interfaces.
+
```

Comment:
Add a paragraph or section about terminology. In particular, the word "interface" is a cause of confusion as it has a different meaning in Profinet. In Profinet terminology, br0, eth0 and eth1 will together form an "interface". br0 will be the "management port" while eth0 and eth1 will be "physical ports".

Reply:
I have tried to clarify this.